miden/src/tools/mod.rs

```diff
@@ -358,19 +359,15 @@ mod tests {
         let host = DefaultHost::default();
         let execution_details =
             super::analyze(&program, stack_inputs, host, Arc::new(DefaultSourceManager::default()));
-        let expected_error = "Execution Error: DivideByZero(1)";
+        let expected_error = "execution error";
         assert_eq!(execution_details.err().unwrap().to_string(), expected_error);
     }
```

Comment:
Those 2 error tests are outdated - they only check the `Display` output of errors, whereas we now have a bunch of tests that check that the `miette`'s `Diagnostic` output is correct. So I would just remove them